### ISSUE TITLE
games-board/q5Go-2.1.3 fails to build

### ISSUE THREAD

**Issue by ilinray** [jl1990/gentoo-overlay#1]
I think that the problem is that dev-qt/qtmultimedia::gentoo package now has version 6.7.1 which might be from Qt6 and not Qt5.

**Comment by ilinray** [comment]
There is still Qt5 version of the dependency in 'gentoo' overlay -- qtmultimedia-5.15.14. I'm not really sure how to modify the ebuild myself though

**Comment by jl1990** [comment]
Hi,

Unfortunately I am not using gentoo at this moment. But specifying a version for qtmultimedia should be easy. I created a branch with the [changes](https://github.com/jl1990/gentoo-overlay/compare/master...fix_qtmultimedia) you would need to include.

Although you need to probably update the ebuild manifest as specified here: https://devmanual.gentoo.org/general-concepts/manifest/

**Comment by ilinray** [comment]
Ty for the fix.
Since your overlay is in "the official list of overlays for everyone to use" (tm), it would be nice if you merged it into master. If I understand portage correctly, I think that would fix the issue for everyone going forward.

**Comment by jl1990** [comment]
The problem is that manifest needs to be updated also, and as far as I know, you need Gentoo for that (and I don't have it installed currently).
If I merge the changes without updating the manifest is going to be broken.

**Comment by ilinray** [comment]
I cloned and switched to the branch with the fix. Then I ran the `ebuild q5Go-2.1.3.ebuild manifest` command and decided to look at the diff. There was no difference with the /var/db/repos/jl1990/games-board/q5Go/Manifest. I just copied the ebuild into /var/db/repos/jl1990/games-board/q5Go and was able to emerge the app from scratch (I scraped all dependencies beforehand too). Portage installed the correct qtmultimedia. That leads me to believe that Manifest doesn't need the update as there is no update.
